app/questions.py

```diff
@@ -1,28 +1,56 @@
-import sys
-from sympy import symbols, latex, Eq, Rational
+from sympy import symbols, latex, Eq
 
-QUESTIONS = {
-    'mcat': set(
-        '1',
-    ),
-}
+class QuestionError:
+    def __init__(self):
+        self.description = "Question not found."
+
+
+class QuestionTypeError:
+    def __init__(self):
+        self.description = "Question type not found."
+
+
+class Question:
+    def __init__(self, seed):
+        self.description = ""
+        self.question = ""
+        self.seed = seed
+    
+    @property
+    def __dict__(self):
```

Comment:
I vote this should be called something nicer

Reply:
something like `json(self)`?